src/PaymentGateways/Gateways/Stripe/Actions/AddStripeAttributesToNewForms.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\Actions;
+
+
+use Give\DonationForms\Models\DonationForm;
+
+class AddStripeAttributesToNewForms
+{
+
+    /**
+     * Add Stripe attributes to the PaymentGateways block new forms.
+     */
+    public function __invoke(DonationForm $form)
+    {
+        $block = $form->blocks->findByName('givewp/payment-gateways');
+        if ($block) {
```

Comment:
I think we would want to make sure Stripe is enabled in global settings before setting this attribute.

Reply:
Sure

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/1741124d0c04c863b0174211cf82fb2a162e7887